sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -613,6 +608,62 @@ def set_scalar(accumulator, metric_update):
   metric_update.scalar = to_json_value(accumulator.value, with_type=True)
 
 
+class CounterUpdateSetters(object):
+  """Setters for translating between accumulators and dataflow counters."""
```

Comment:
What is the relationship between "dataflow counters" and "metric updates"?  This comment refers to counters, while the code below refers to metric update protos.  Should this comment instead mention translating accumulators for metric updates so that this class should be renamed accordingly?

Also, the naming of the methods below is confusing: two arguments are passed, e.g. set_boolean(accumulator, metric_update_proto).  Just reading this signature suggests that the accumulator will somehow have a boolean set from metric_update_proto, which is the opposite of what is intended.  Can you instead rename these methods translate_*, e.g. for translate_boolean(accumulator, metric_update_proto)?  This would make it clear that we are translating accumulator into metric_update_proto.  If you do this, can you rename set_scalar and set_mean in this file as well?